agent/stats/engine.go

```diff
@@ -390,6 +390,27 @@ func (engine *DockerStatsEngine) containerHealthsToMonitor() bool {
 	return len(engine.tasksToHealthCheckContainers) != 0
 }
 
+func (engine *DockerStatsEngine) stopTrackingContainer(container *StatsContainer, taskARN string) bool {
```

Comment:
Can you also add documentation for this method explaining what it is that method is returning?